components/OpenIbox.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import StyledButton from './StyledButton';
+
+function OpenEmailProviderButton({ email }) {
+  const gmailregex = /@gmail\.com$/.test(email);
+  const hotmailregex = /@outlook\.com$/.test(email);
+  const gmaillink =
+    'https://mail.google.com/mail/u/2/#advanced-search/subject=Open+Collective%3A+Login&amp;subset=all&amp;within=1d';
+  const hotmaillink = 'https://outlook.live.com/mail/inbox';
+  const [btntext] = useState(gmailregex ? 'Gmail' : hotmailregex ? 'HotMail' : 'Inbox');
+  const [link] = useState(gmailregex ? gmaillink : hotmailregex ? hotmaillink : '');
+  const [emailProvider] = useState(gmailregex || hotmailregex ? true : false);
+
+  const handleClick = () => {
+    window.open(link);
+  };
+
+  return emailProvider ? (
+    <StyledButton
+      data-cy="signin-btn"
+      buttonStyle="primary"
+      fontWeight="600"
+      minWidth={200}
+      ml={3}
+      onClick={handleClick}
+    >
+      Open {btntext}
+    </StyledButton>
+  ) : (
+    ''
```

Comment:
It's good to return `null` when we don't want to render anything

```suggestion
    null
```